File: caoguodong/Excel Sheet Column Title.java
public class Solution {
    public String convertToTitle(int n) {
        String result = "";
        if(n>26){
            int i = n%26==0?n/26-1:n/26;
            result += convertToTitle(i);
            n=n-26*i;
        }
        result += String.valueOf((char)(n+64));
        return result;
    }
}

File: lizhiqiang/Excel Sheet Column Title.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string result;;
        while (n > 0) {
            int posNum = (n - 1) % 26;
            char pos = 'A' + posNum;
            result.push_back(pos);
            n = (n - 1) / 26;
        }
        reverse(result.begin(), result.end());
        return result;
    }
};
